### DISCUSSION TITLE
load Unsplash pictures in the navbar component (brand url)?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#121] [Q&A] [answered]
Hi Renaud,

I see that all navbar components brand pictures in your example apps are  loaded from an \assets folder. But is it possible to load single images  from Unsplash?

Kind regrads
Folkert

**Comment by renaudpawlak** [comment, marked as answer]
As long as you have a URL (absolute URL) to your image, you should be able to use it in the brand property of the navbar. If you don't have a URL, you can encode your image as a base64 URL and paste it in the brand field. However, it is not recommended because it makes you app model heavy and disable the browser's cache capabilities.

**Reply by HFolkertH** [reply]
Thanhs. It works now.
